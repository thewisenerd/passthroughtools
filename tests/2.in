# The following is the parsable format, which can be fed to other
# programs. Each different item in every column has an unique ID
# starting from zero.
# CPU,Core,Socket,Node,,L1d,L1i,L2,L3
0,0,0,0,,0,0,0,0
1,1,0,0,,1,1,1,0
2,2,0,0,,2,2,2,1
3,3,0,0,,3,3,3,1
4,4,0,0,,4,4,4,2
5,5,0,0,,5,5,5,2
6,6,0,0,,6,6,6,3
7,7,0,0,,7,7,7,3
8,8,0,0,,8,8,8,4
9,9,0,0,,9,9,9,4
10,10,0,0,,10,10,10,5
11,11,0,0,,11,11,11,5
12,12,0,0,,12,12,12,6
13,13,0,0,,13,13,13,6
14,14,0,0,,14,14,14,7
15,15,0,0,,15,15,15,7
16,16,0,0,,16,16,16,8
17,17,0,0,,17,17,17,8
18,18,0,0,,18,18,18,9
19,19,0,0,,19,19,19,9
20,20,0,0,,20,20,20,10
21,21,0,0,,21,21,21,10
22,22,0,0,,22,22,22,11
23,23,0,0,,23,23,23,11
24,24,0,0,,24,24,24,12
25,25,0,0,,25,25,25,12
26,26,0,0,,26,26,26,13
27,27,0,0,,27,27,27,13
28,28,0,0,,28,28,28,14
29,29,0,0,,29,29,29,14
30,30,0,0,,30,30,30,15
31,31,0,0,,31,31,31,15
32,32,1,1,,32,32,32,16
33,33,1,1,,33,33,33,16
34,34,1,1,,34,34,34,17
35,35,1,1,,35,35,35,17
36,36,1,1,,36,36,36,18
37,37,1,1,,37,37,37,18
38,38,1,1,,38,38,38,19
39,39,1,1,,39,39,39,19
40,40,1,1,,40,40,40,20
41,41,1,1,,41,41,41,20
42,42,1,1,,42,42,42,21
43,43,1,1,,43,43,43,21
44,44,1,1,,44,44,44,22
45,45,1,1,,45,45,45,22
46,46,1,1,,46,46,46,23
47,47,1,1,,47,47,47,23
48,48,1,1,,48,48,48,24
49,49,1,1,,49,49,49,24
50,50,1,1,,50,50,50,25
51,51,1,1,,51,51,51,25
52,52,1,1,,52,52,52,26
53,53,1,1,,53,53,53,26
54,54,1,1,,54,54,54,27
55,55,1,1,,55,55,55,27
56,56,1,1,,56,56,56,28
57,57,1,1,,57,57,57,28
58,58,1,1,,58,58,58,29
59,59,1,1,,59,59,59,29
60,60,1,1,,60,60,60,30
61,61,1,1,,61,61,61,30
62,62,1,1,,62,62,62,31
63,63,1,1,,63,63,63,31
