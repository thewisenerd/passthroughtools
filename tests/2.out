  <vcpu placement='static'>8</vcpu>
  <cputune>
    <vcpupin vcpu='0' cpuset='56'/>
    <vcpupin vcpu='1' cpuset='57'/>
    <vcpupin vcpu='2' cpuset='58'/>
    <vcpupin vcpu='3' cpuset='59'/>
    <vcpupin vcpu='4' cpuset='60'/>
    <vcpupin vcpu='5' cpuset='61'/>
    <vcpupin vcpu='6' cpuset='62'/>
    <vcpupin vcpu='7' cpuset='63'/>
  </cputune>
  <cpu mode='host-passthrough' check='none'>
    <topology sockets='1' cores='8' threads='1'/>
    <cache mode='passthrough'/>
  </cpu>
